cypress/pageObjects/basket.page.js

```diff
@@ -0,0 +1,12 @@
+class BasketPage {
+  elements = {
+    productItemRow: () => cy.get('mat-row'),
+    deleteButton: () => ('.cdk-column-remove'),
+  }
+
+  deleteProductFromBasketByIndex (productIndex) {
+    this.elements.productItemRow().eq(productIndex).find(this.elements.deleteButton()).click()
```

Comment:
Is there any other way to define the deleteButton and not use find(). For further discussion